Impl budget report reviewer page

malloc3141 opened the pull request:
# 요약 \*

<!-- 닫는 이슈 번호 및 PR 내용에 대한 간단한 요약 표기. -->

It closes #129 

# 스크린샷

<!-- PR 변경 사항에 대한 스크린샷이나 .gif 파일 -->
![image](https://github.com/user-attachments/assets/43333978-7e3d-4e5f-8203-b1d4a5acfa5c)


# 이후 Task \*

<!-- PR 이후 개설할 이슈 목록 -->

- 없음

malloc3141 committed:
feat: impl budget report reviewer page

ChaeyeonAhn reviewed (commented):
이거 혹시 Table 컴포넌트 `width: "fit-content"`로 수정해서 그냥 테이블 짧으면 짧은대로 나오도록 해주실 수 있으신가요?? 앞으로 그냥 이케 할라궁

ChaeyeonAhn reviewed (commented):
나머진 좋아보여요! 굿굿 넘넘 멋져. 👍 👍 👍 👍 
테이블만 수정하구 알려주세여!!

malloc3141 committed:
merge with remote
Merge branch 'dev' of github.com:sparcs-kaist/students into 129-impl-budget-report-reviewer

malloc3141 committed:
fix: fix table width to fit-content

ChaeyeonAhn reviewed (approved):
고고
